### ISSUE TITLE
Can't mark telescope.lua

### ISSUE THREAD

**Issue by gldtn** [dharmx/track.nvim#11]
I just noticed, I can mark any files as intended within my project, but for some reason telescope.lua is not getting marked.

Here is my telescope config: https://github.com/gldtn/dotfiles/blob/main/.config/nvim/lua/plugins/telescope.lua
track.lua config: https://github.com/gldtn/dotfiles/blob/main/.config/nvim/lua/plugins/track.lua

No matter how many times I `:Mark` telescope.lua it does not show up on the list like the other files:

![tele_notshowing](https://github.com/dharmx/track.nvim/assets/981383/e5dd61c0-feed-462f-9c94-6b581810572c)

This one seems a bit weird, any ideas as to why it might be happening?

**Comment by dharmx** [comment]
Yes, it does seem weird. Could you try setting the log level to debug and then send the ~/.cache/nvim/track.log here?

Also, see if the `:lua=require'track.state'._roots[vim.loop.cwd()]:get_main_bundle().views` actually changes after you add the mark.

**Comment by gldtn** [comment]
Hopefully I did it correctly, I set inside track.lua setup function:
```lua
log = {
				plugin = "track",
				level = "debug",
			},
```

Here is the log from `~/.cache/nvim/track.log`
```log
[INFO  Sun Apr 28 00:40:02 2024] ~/.local/share/nvim/lazy/track.nvim/lua/track/state.lua:156: State.save(): saved current state into ~/.local/state/nvim/track.json
[INFO  Sun Apr 28 00:40:02 2024] ~/.local/share/nvim/lazy/track.nvim/lua/telescope/_extensions/track/pickers/views.lua:69: Telescope.Views.picker(): closed telescope.track.views and saved state
```

Notification message when I run `:Mark` on telescope.lua
```log
 | Messages:  | Messages:  | Messages:  | Messages:  | Messages: [track] [INFO  00:33:24] ~/.local/share/nvim/lazy/track.nvim/lua/track/state.lua:129: State.reload(): reloaded ~/.local/state/nvim/track.json
```
Notification message from `:lua=require'track.state'._roots[vim.loop.cwd()]:get_main_bundle().views`
```log
 | Messages:  | Messages:  | Messages: { "lua/plugins/track.lua", "lua/core/keymaps.lua", "~/.config/nvim/lua/plugins/lsp/lspconfig.lua", "lua/core/options.lua",
  <metatable> = {
    __call = <function 1>
  }
}
```

**Comment by dharmx** [comment]
Thank you. Unfortunately it seems that the logs are useless :disappointed:.
I can mark your file in mine though.

Anyways, you should try manually passing in the name for testing purposes.

```vim
" cd into ~/.config/nvim
:Mark lua/plugins/telescope.lua
:Track
:lua=require'track.state'._roots[vim.loop.cwd()]:get_main_bundle().views
```

**Comment by gldtn** [comment]
No luck with doing it manually. Funny thing is. if I rename the file to anything other than `telescope` itself.. for example `telescopetj.lua`, then it gets marked. 😕 

Probably something fishy going with my config 🤷‍♂️

**Comment by gldtn** [comment]
I remembered I was trying to clear and delete all marks from neovim a few days ago, which might of caused this to happen. So I believe it "corrupted" shada somehow?

I. ran `:delm! | delm A-Z0-9`. Not sure if this is the exact cause but it's the only thing I can think off. I just deleted the following folders and restarted neom and track is marking the telescope.lua file as it should.

```bash
rm -rf ~/.local/share/nvim
rm -rf ~/.local/state/nvim
rm -rf ~/.cache/nvim
```

Probably just the `share/nvim` would of been enough, but anyways, it worked.

Thanks again for taking your time and helping me solve this issue!
